nixos/modules/virtualisation/qemu-vm.nix

```
@@ -139,6 +159,8 @@ let
 
   # Generate a hard disk image containing a /boot partition and GRUB
   # in the MBR.  Used when the `useBootLoader' option is set.
+  # Uses `runInLinuxVM` to the image in a throwaway VM.
```

I think a verb is missing here:

```suggestion
  # Uses `runInLinuxVM` to build the image in a throwaway VM.
```

---

Thanks, fixed.